exam1/exam1/Program.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace exam
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            int upperBorder = 10000000;
```

Comment:
В задании 10 млрд, у вас 10 млн
```suggestion
           long upperBorder = 10000000000;
```